### PR TITLE
GPG signing key updated for maven central/sonatype artifact repositories

### PR DESCRIPTION
**Pull request by indyaah**
Ref: https://github.com/CitrineInformatics/lolo/pull/183

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: java
 jdk:
 - oraclejdk8
-install: mvn install -P !build-extras -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+install: mvn install -P !build-extras -DskipTests=true -Dmaven.javadoc.skip=true -B
+  -V
```
Comment (sparadiso) [inline .travis.yml]: Why the newline?

**Review by sparadiso — DISMISSED** [review]
LGTM - why the newline, though?

File: .travis.yml (hunk 1 above)
Comment (maxhutch) [reply .travis.yml]: travis auto-reformats

**Review by maxhutch — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (indyaah) [reply .travis.yml]: Fixed :)

**Review by sparadiso — APPROVED** [review]
LGTM

**Pull request merged**